ktor-server/ktor-server-plugins/ktor-server-content-negotiation/jvm/src/DefaultIgnoredTypesJvm.kt

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.plugins.contentnegotiation
+
+import io.ktor.http.*
+import io.ktor.utils.io.*
+import java.io.*
+import kotlin.reflect.*
+
+internal actual val DefaultIgnoredTypes: Set<KClass<*>> =
+    mutableSetOf(HttpStatusCode::class, ByteReadChannel::class, InputStream::class)
```

Comment:
Should we also ignore `ByteArray`, `ByteBuffer` and etc?

Reply:
In a server transforming to/from `ByteArray` happens before `ContentNegotiation`